src/lib.rs

```diff
@@ -480,20 +486,23 @@ impl Inotify {
     ///
     /// An internal buffer which can hold the maximum possible size is used.
     pub fn event_stream(&mut self) -> EventStream {
-        EventStream::new(self.fd.clone(), false)
+        EventStream::new(self.fd.clone())
     }
 
     /// Transform `self` into a `Stream` of events.
     ///
-    /// This behaves the same as [`event_stream`], but unlike that function,
-    /// the file descriptor will be closed when the _stream_ is dropped, rather
-    /// than when the `Inotify` instance that produced it is dropped. This
-    /// should be used when the `EventStream` should live longer than the
-    /// `Inotify`, such as when it is returned from a function.
+    /// This behaves the same as [`Inotify::event_stream`]. Since the file
+    /// descriptor management issues that necessitated this function have
+    /// been fixed, it should no longer be necessary, and
+    /// [`Inotify::event_stream`] should be used instead.
     ///
-    /// [`event_stream`]: struct.Inotify.html#method.event_stream
+    /// [`Inotify::event_stream`]: struct.Inotify.html#method.event_stream
+    #[deprecated(
+        since = "0.5.2",
+        note = "use [`Inotify::event_stream`](struct.Inotify.html#method.event_stream) instead."
+    )]
```

Comment:
I think this deprecation is unnecessary, as this method hasn't been released yet (it only exists in the repository). Would you mind removing it instead?

Reply:
Will do. I wasn't sure what your preferred rules around depredations are, so I figured I'd make the more conservative choice here (also, wanted to avoid breaking my own build :P).